openAPI/api.yaml

```diff
@@ -72,14 +72,8 @@ components:
       type: object
       properties:
         vtr:
-          type: array
-          items:
-            type: string
-            enum:
-              - P1
-              - P2
-              - P3
-              - P4
+          type: string
+          pattern: "^P[1-4](,P[1-4])*$"
```

Comment:
I assume this pattern works for if there are more than 2 PXs? That's what the `*` is for right?

Reply:
Ah ok, maybe just squash it all into a single commit? There's not a huge amount of change overall and a lot more change across the 5 commits that overall gets changed/reverted. Would squashing the whole lot work?